includes/definitions/ptp500.yaml

```diff
@@ -0,0 +1,11 @@
+os: ptp500
+text: Cambium PTP 300/500
+type: wireless
+icon: cambium
+mib_dir: cambium\500
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
```

Comment:
You can add some of the wireless graphs here to make it a little more informative for users. Max 4 I think.